src/fileActions.jsx

```diff
@@ -454,12 +452,10 @@ export const EditPermissionsModal = ({ selected, path }) => {
     return (
         <Modal
           position="top"
-          variant="medium"
-          title={selected.type === "file"
-              ? _("File properties and access")
-              : selected.type === "link"
-                  ? _("Link properties and access")
-                  : _("Directory properties and access")}
+          variant={ModalVariant.small}
+          /* Translators: $0 represents a filename */
+          title={cockpit.format(_("“$0” permissions"), selected.name)}
+          description={inode_types[selected.type] || "Unknown type"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-194-20240123-171128-953511af-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)